srv/salt/ceph/tests/symlink/init.sls

```diff
@@ -0,0 +1,29 @@
+
+# Find disk of first OSD
+{% set osd_id = salt['osd.list']() %}
+{% set disk = salt['osd.device'](osd_id[0]) %}
```

Comment:
@swiftgist This module `osd.device` appears not to exist. Did you mean `osd.devices` or `osd.osd_device`?